metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -302,7 +302,6 @@ void convert_batch_to_row_diff(const std::string &pred_succ_fprefix,
 
     const std::filesystem::path tmp_path = utils::create_temp_dir(
             std::filesystem::path(dest_dir).remove_filename(), "col");
-    logger->trace("Using temporary directory {}", tmp_path);
```

Comment:
Instead of removing this, why not wrap it around an `#ifndef NDEBUG ... #endif` block instead?

Reply:
`utils::create_temp_dir` already prints a line in the log that it created a new temp dir, so I figured this second message is redundant.